spec/factories/task.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :task do
+    name { Faker::Lorem.sentence }
```

Comment:
Usually, the name of the task consists of one single word or sometimes multiple in the snake case format.